Apps/frontend/src/views/OnOffButton.svelte

```diff
@@ -10,6 +10,10 @@
     isOn = !isOn;
     dispatch("switch-plot-enabled", { enabled: isOn });
   };
+
+  export const click = () => {
```

Comment:
Just a minor pet peeve of mine. I'd prefer `onClick` to indicate it is an `on[Whatever]`-function. YOu don't have to change it :)

Reply:
But this is not an ON-click handler. It is more a DO-click as it is supposed to be used for simulating the click from outside. I can rename it anyways and I share your preferences regarding the naming but in this case I chose that name intentionally.

Reply:
I see where you're coming from. Then I have another quesiton, why not call `handleClick` directly?